### PR TITLE
feat: added a notification panel

### PR DESCRIPTION
**Pull request by raizo07**
- [x] issue https://github.com/keep-starknet-strange/art-peace/issues/182
- [x] follows contribution [guide](https://github.com/keep-starknet-strange/art-peace/blob/main/CONTRIBUTING.md)
- [x] code change includes tests

### CODE REVIEW

**Review by b-j-roberts — COMMENTED** [review]
I'll take a closer look soon, but in the meantime, can you remove the toastify module and code since it seems you aren't using it anymore?

**Comment by raizo07** [comment]
> I'll take a closer look soon, but in the meantime, can you remove the toastify module and code since it seems you aren't using it anymore?

@b-j-roberts Done. I've removed them.

**Comment by b-j-roberts** [comment]
> @b-j-roberts Done. I've removed them.

Cool, could you revert the changes to `package-lock.json` too?

File: frontend/src/tabs/NotificationPanel.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.notification-panel {
```
Comment (b-j-roberts) [inline frontend/src/tabs/NotificationPanel.css:1]: Can you please match this to make the styling like the other panels : 
https://github.com/keep-starknet-strange/art-peace/blob/b7122c3eedf5e2be34ea392c189e8b15d4d2f84e/frontend/src/tabs/BasicTab.css#L1

File: frontend/src/tabs/NotificationPanel.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  }
+  
+  .notification-content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: #000000E6;
+  }
+  
+  .notification-content button {
```
Comment (b-j-roberts) [inline frontend/src/tabs/NotificationPanel.css:30]: There are styles here for close buttons, to make them match the style of the other panels : 
https://github.com/keep-starknet-strange/art-peace/blob/b7122c3eedf5e2be34ea392c189e8b15d4d2f84e/frontend/src/tabs/BasicTab.css#L27
https://github.com/keep-starknet-strange/art-peace/blob/b7122c3eedf5e2be34ea392c189e8b15d4d2f84e/frontend/src/utils/Styles.css#L99

**Review by b-j-roberts — CHANGES_REQUESTED** [review]
Some work still needs to be done on the styling to make it match the rest of the app panels.

Also, there are changes in `frontend/package-lock.json` and `package-lock.json` that should be removed.

**Comment by raizo07** [comment]
@b-j-roberts I've fixed them, kindly review.

**Pull request merged**